src/test/app/claims/models/claim.ts

```diff
@@ -1233,3 +1235,17 @@ describe('OconFormResponse', () => {
     expect(claim.status).to.be.equal(ClaimStatus.DEFENDANT_OCON_FORM_RESPONSE)
   })
 })
+
+describe('ScannedDocument', () => {
+  it('should return Claim Documents including Scanned Document', () => {
+    const claimWithResponse = new Claim().deserialize({ ...claimStoreMock.sampleClaimIssueObj, ...claimStoreMock.sampleClaimDocuments })
+    const claimDocs: ClaimDocument[] = claimWithResponse.claimDocuments
+    expect(2).to.be.eq(claimDocs.length)
+  })
+  it('should return Claim Documents including Scanned Document', () => {
+    claimStoreMock.sampleClaimDocuments.claimDocumentCollection.claimDocuments = undefined
+    const claimWithResponse = new Claim().deserialize({ ...claimStoreMock.sampleClaimIssueObj, ...claimStoreMock.sampleClaimDocuments })
+    const claimDocs: ClaimDocument[] = claimWithResponse.claimDocuments
+    expect(1).to.be.eq(claimDocs.length)
```

Comment:
If there are no Claim documents how would the length be 1?

Reply:
I think i got the answer (its the scanned document)

Reply:
yes correct